src/pages/documentation/edge-network/tabs/index.md

```diff
@@ -10,11 +12,7 @@ implementation 'com.adobe.marketing.mobile:edge:2.+'
 implementation 'com.adobe.marketing.mobile:edgeidentity:2.+'
 ```
 
-<InlineNestedAlert variant="warning" header="false" iconPosition="left">
-
-Using dynamic dependency versions is **not** recommended for production apps. Please read the [managing Gradle dependencies guide](../resources/manage-gradle-dependencies.md) for more information. 
-
-</InlineNestedAlert>
+<Alerts query="platform=android-gradle&componentClass=InlineNestedAlert"/>
```

Comment:
uuuu, can we use this everywhere now? 😍
and can we have something similar for the registerExtension API and use for all extensions?

Reply:
This is a thing we can do????? That's actually so cool LOL

Reply:
Yeah that makes a lot of sense then. If this works on localhost, I'm satisfied with that, and 100% makes things SO much easier for maintaining.

Reply:
@timkimadobe For variant tagging, yes, let's take that on Slack to discuss what the best way to do that will be. We can then add that to the repo's readme.